### ISSUE TITLE
Cannot use MenuItem with RN 0.62

### ISSUE THREAD

**Issue by pnthach95** [mxck/react-native-material-menu#86]
```
TypeError: undefined is not an object (evaluating '_reactNative.TouchableHighlight.propTypes.style')
```

In RN 0.62, they removed propTypes from TouchableHighlight, so it broke.

**Comment by MujtabaFR** [comment]
I'm having the same issue with **RN 0.62** and **react-native-material-menu 1.0.0**
still looking for a solution

**Comment by mxck** [comment]
I published 1.1.0 version without PropTypes at all :)
